engine/src/main/java/org/terasology/logic/players/MenuControlSystem.java

```diff
@@ -64,6 +85,31 @@ public void onScreenshotCapture(ScreenshotButton event, EntityRef entity) {
         }
     }
 
+    @Command(shortDescription = "Triggers the creation of a save game", runOnServer = true,
+            requiredPermission = PermissionManager.SERVER_MANAGEMENT_PERMISSION)
+    @ReceiveEvent(components = {ClientComponent.class})
+    public void onSave(SaveButton event, EntityRef entity) {
+        if (event.getState() == ButtonState.DOWN) {
+            GameManifest latestManifest = getLatestGameManifest();
+            storageManager.setSavePath(PathManager.getInstance().getSavePath(latestManifest.getTitle()).resolve(latestManifest.getTitle() + " Quick Save"));
+            storageManager.requestSaving();
+            event.consume();
+        }
+    }
+
+    @ReceiveEvent(components = {ClientComponent.class})
+    public void onLoad(LoadButton event, EntityRef entity) {
+        if (event.getState() == ButtonState.DOWN) {
+            GameManifest quickSaveGameManifest = getQuickSaveGameManifest();
+            if (quickSaveGameManifest != null) {
+                boolean saveRequested = false; //quick load shouldn't cause the game to save
+                boolean isQuickLoad = true;
+                CoreRegistry.get(GameEngine.class).changeState(new StateLoading(quickSaveGameManifest, NetworkMode.NONE, isQuickLoad), saveRequested);
```

Comment:
CoreRegistry is deprecated. Use an `@In` annoated field of the wanted type instead. Alternativly you can if available use a Context object instance to get a value.